### ISSUE TITLE
Failing to open pickle file from an umbrella run

### ISSUE THREAD

**Issue by vpalaciob-pme** [SSAGESLabs/PySAGES#266]
This is the stack trace: 
```
File [~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/pysages/methods/core.py:81](~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/pysages/methods/core.py:81), in SamplingMethod.__setstate__(self, state)
     80 def __setstate__(self, state):
---> 81     default_setstate(self, state)

File [~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/pysages/methods/core.py:282](~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/pysages/methods/core.py:282), in default_setstate(method, state)
    280 def default_setstate(method, state):
    281     args, kwargs = state
--> 282     method.__init__(**args, **kwargs)

File [~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/plum/function.py:642](~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/plum/function.py:642), in _BoundFunction.__call__(self, *args, **kw_args)
    641 def __call__(self, *args, **kw_args):
--> 642     return self.f(self.instance, *args, **kw_args)

File [~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/plum/function.py:592](~/miniforge3/envs/hoomdtest/lib/python3.8/site-packages/plum/function.py:592), in Function.__call__(self, *args, **kw_args)
    590 # Cache failed. Get method and return type, and perform call.
    591 method, return_type = self.resolve_method(*sig_types)
--> 592 return _convert(method(*args, **kw_args), return_type)

TypeError: __init__() missing 1 required positional argument: 'cvs'
```

**Comment by pabloferz** [comment]
I think this should have been fixed in #257. Can you confirm running with the latest PySAGES version @vpalaciob-pme?

**Comment by pabloferz** [comment]
I think this should not happen anymore. Feel free to reopen if it keeps happening.
